src/main/java/io/cryostat/rules/RuleRegistry.java

```diff
@@ -179,9 +179,18 @@ public void deleteRules(ServiceRef serviceRef) throws IOException {
         }
     }
 
+    public void enableRule(Rule rule, boolean enabled) {
+        String name = rule.getName();
+        if ((enabled && !rule.isEnabled()) || (!enabled && rule.isEnabled())) {
```

Comment:
Is this equivalent to `if (enabled != rule.isEnabled())`?

Reply:
Yes >.<